scripts/ioc_deploy.py

```diff
@@ -203,9 +209,65 @@ def finalize_name(name: str, github_org: str, verbose: bool) -> str:
             raise ValueError(
                 f"Error cloning repo, make sure {name} exists in {github_org} and check your permissions!"
             ) from exc
+        try:
+            with open(Path(tmpdir) / name / "README.md", "r") as fd:
```

Comment:
This is fair, I could be more permissive with the kinds of files to check. I originally wanted to avoid answering the question "what do I do when there are two readme files?"

Reply:
The answer I'm picking is "include them all in some arbitrary order and do our best"